plugin/src/main/java/net/dzikoysk/funnyguilds/listener/PlayerChat.java

```diff
@@ -28,174 +32,113 @@ public void onChat(AsyncPlayerChatEvent event) {
         }
         User user = userOption.get();
 
-        if (user.hasGuild()) {
-            Guild guild = user.getGuild().get();
-            String message = event.getMessage();
+        boolean isGuildChat = user.getGuild()
+                .map(guild -> this.sendGuildMessage(player, guild, event.getMessage()))
+                .orElseGet(false);
 
-            if (sendGuildMessage(event, message, player, guild)) {
-                if (config.logGuildChat) {
-                    FunnyGuilds.getPluginLogger().info("[Guild Chat] " + message);
-                }
+        if (isGuildChat) {
+            event.setCancelled(true);
 
-                return;
+            if (config.logGuildChat) {
+                FunnyGuilds.getPluginLogger().info("[Guild Chat] " + event.getMessage());
             }
+
+            return;
         }
 
         int points = user.getRank().getPoints();
-        String format = event.getFormat();
 
-        format = StringUtils.replace(format, "{RANK}", StringUtils.replace(config.chatRank.getValue(), "{RANK}", String.valueOf(user.getRank().getPosition(DefaultTops.USER_POINTS_TOP))));
-        format = StringUtils.replace(format, "{POINTS}", config.chatPoints.getValue());
-        format = StringUtils.replace(format, "{POINTS-FORMAT}", NumberRange.inRangeToString(points, config.pointsFormat));
-        format = StringUtils.replace(format, "{POINTS}", String.valueOf(points));
+        Formatter formatter = new Formatter()
+                .register("{RANK}", StringUtils.replace(config.chatRank.getValue(), "{RANK}", String.valueOf(user.getRank().getPosition(DefaultTops.USER_POINTS_TOP))))
+                .register("{POINTS}", config.chatPoints.getValue())
+                .register("{POINTS-FORMAT}", NumberRange.inRangeToString(points, config.pointsFormat))
+                .register("{POINTS}", String.valueOf(points));
 
-        if (user.hasGuild()) {
-            format = StringUtils.replace(format, "{TAG}", StringUtils.replace(config.chatGuild.getValue(), "{TAG}", user.getGuild().get().getTag()));
-            format = StringUtils.replace(format, "{POS}", StringUtils.replace(config.chatPosition.getValue(), "{POS}", getPositionString(this.userManager.findByPlayer(player))));
-        }
-        else {
-            format = StringUtils.replace(format, "{TAG}", "");
-            format = StringUtils.replace(format, "{POS}", "");
-        }
+        user.getGuild()
+                .peek(guild -> {
+                    formatter.register("{TAG}", StringUtils.replace(config.chatGuild.getValue(), "{TAG}", user.getGuild().get().getTag()));
```

Comment:
user.getGuild() here is cringe

Reply:
`user.getGuild().get().getTag()` -> `guild.getTag()` ?